buildtest/cli/__init__.py

```diff
@@ -1109,28 +1110,43 @@ def report_menu(subparsers, parent_parser):
         help="Filter report by filter fields. The filter fields must be a key=value pair and multiple fields can be comma separated in the following format: --filter key1=val1,key2=val2 . For list of filter fields run: --helpfilter.",
     )
 
-    filter_group.add_argument(
-        "--format",
-        help="format field for printing purposes. For more details see --helpformat for list of available fields. Fields must be separated by comma (usage: --format <field1>,<field2>,...)",
-    )
     filter_group.add_argument(
         "--helpfilter",
         action="store_true",
         help="List available filter fields to be used with --filter option",
     )
-    filter_group.add_argument(
-        "--helpformat", action="store_true", help="List of available format fields"
-    )
+
     filter_group.add_argument(
         "--filterfields",
         action="store_true",
         help="Print raw filter fields for --filter option to filter the report",
     )
-    filter_group.add_argument(
+
+    format_group = parser_report.add_argument_group("format", "Format options")
+    format_detailed_group = parser_report.add_mutually_exclusive_group()
+
+    format_detailed_group.add_argument(
+        "--format",
+        help="format field for printing purposes. For more details see --helpformat for list of available fields. Fields must be separated by comma (usage: --format <field1>,<field2>,...)",
+    )
+
+    format_group.add_argument(
+        "--helpformat", action="store_true", help="List of available format fields"
+    )
+
+    format_group.add_argument(
         "--formatfields",
         action="store_true",
         help="Print raw format fields for --format option to format the report",
     )
```

Comment:
declare these arguments after declaration of argument `format_group` but before declaration of the following

```python
format_detailed_group = parser_report.add_mutually_exclusive_group()
```

This helps in reading code where options are declared near the parser declaration.